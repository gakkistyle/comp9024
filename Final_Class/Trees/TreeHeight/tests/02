6 3
3 6
