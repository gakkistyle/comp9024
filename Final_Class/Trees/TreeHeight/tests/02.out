Enter the preorder traversal of the tree: Enter the in-order traversal of the tree: Tree:

  6
 /
3

TreeHeight returned: 1
